JPS_Ontology/KBTemplates/TBox/OntoSewage.csv

```diff
@@ -0,0 +1,169 @@
+Source,Type,Target,Relation,Domain,Range,Quantifier,Comment,Defined By,Label
+OntoSewage,TBox,https://www.theworldavatar.com/kg/ontosewage/,https://www.w3.org/2007/05/powder-s#hasIRI,,,,,,
+OntoSewage,TBox,1,http://www.w3.org/2002/07/owl#versionInfo,,,,,,
+OntoSewage,TBox,OntoSewage is an ontology developed for representing sewerage system.,http://www.w3.org/2000/01/rdf-schema#comment,,,,,,
+OntoSewage,TBox,,http://www.w3.org/2002/07/owl#imports,,,,,,
+SewerageComponent,Class,,,,,,"is the infrastructure that conveys sewage or surface runoff (stormwater, meltwater, rainwater) using sewers.",https://www.theworldavatar.com/kg/ontosewage/,Sewerage component
+SewerageNetwork,Class,,,,,,is a conglomeration of pipes and sewers that work together to collect and drain wastewater from the population or industrial centers to off-site treatment facilities.,https://www.theworldavatar.com/kg/ontosewage/,Sewerage network
+SubNetwork,Class,SewerageNetwork,IS-A,,,,contains branch pipes,https://www.theworldavatar.com/kg/ontosewage/,Sub network
+MainNetwork,Class,SewerageNetwork,IS-A,,,,contains main pipes,https://www.theworldavatar.com/kg/ontosewage/,Main network
+SewerageUsage,Class,,,,,,representing for what the sewerage is used,https://www.theworldavatar.com/kg/ontosewage/,Sewerage usage

```

Comment:
Comments are used very inconsistently, i.e.
- some (sub-)classes do not have any description
- some descriptions (especially the ones starting with verbs) sound odd to me. I suggest changing e.g. "is the type of a manhole." to "Type of manhole"
- as we had many discussions about the precise meaning of particular properties, please try to incorporate all those insights into the comments, so that we do not face those ambiguities again (wherever possible)

Reply:
We tried to write description for as much sub-classes as we could. Some sub-classes such as `40T` or `12.5T` are not clear in definition.
Some of the comments are updated as you suggested.

Reply:
The guiding principle should be that somebody not too familiar with the topic/domain should get a rough idea about a concept by reading those comments. I am (still) not convinved that this is the case for all concepts, e.g. the comment `Being extended` is quite meaningless, especially for the concept `Constructed` as a Functional State. Please revisit the comments and try to be more specific